packages/template-retail-react-app/app/components/forms/useAddressFields.jsx

```diff
@@ -152,13 +152,14 @@ export default function useAddressFields({
                 ...(countryCode === 'CA' ? provinceOptions : stateOptions)
             ],
             rules: {
-                required: formatMessage(
-                    {
-                        defaultMessage: 'Please select your {stateOrProvince}.',
-                        id: 'use_address_fields.error.please_select_your_state_or_province'
-                    },
-                    {stateOrProvince: countryCode === 'CA' ? 'province' : 'state'}
-                )
+                required:
+                    countryCode === 'CA'
+                        ? 'Please select your province.' // FYI we won't translate this
```

Comment:
I think we should create a ticket to remove this ternary operator and make it so that CA has it's own translations file.